lib/diagnostic.rb

```diff
@@ -6,21 +6,21 @@
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = require_relative '../lib/example.rb'
 
```

Comment:
When you hear of running a script, you should think of your command line, where we've been executing Ruby scripts. For that, we've been using `ruby lib/example.rb` from the command line